### ISSUE TITLE
Mixtral 8x22B mixing up syllables

### ISSUE THREAD

**Issue by stefanvarunix** [ggerganov/llama.cpp#6846]
Has anyone experienced this:

I get weird typos in German, e.g. vowels are typed twice ("ii" instead "i", or "aa" instead "a"), mixing up syllables up to writing write gibberish, mainly mixing syllables and letters. The longer the chat (e.g. the context), the worse it gets. In the beginning (e.g. first chat responses), it seems ok.

I downloaded and tested models from https://huggingface.co/MaziyarPanahi (different quants) and https://huggingface.co/mradermacher/Mixtral-8x22B-Instruct-v0.1-GGUF, did GGUF generation and quantization myself etc.
Nothing helped.

The inferencing itself works and is quite fast (Apple M1 Ultra 128 GB 64C GPU).

I use the latest llama.cpp server and API (`./server -m ...`), chat via Panel ChatInterface, accessing the llama.cpp HTTP API with OpenAI's python library.

Is this maybe a problem with tokenizers or chat templates?

**Comment by mirekphd** [comment]
Do you have the same problems (for the same prompts and the same seed) with the previous Mixtral 8x7B? 

We are also seeing lots of typos in the English-langauge output of Mixtral 8x22B - 4-bit quant is practically useless due to these typos and some typos are seen even in the normally nearly lossless 8-bit quant (even though they were not seen in the older smaller Mixtral 8x7B).

**Comment by stefanvarunix** [comment]
Mixtral 8x7B works without these problems.
Mixtral 8x22B has those typos.
I tried different quant providers ([MaziyarPanahi](https://github.com/ggerganov/llama.cpp/issues/MaziyarPanahi/Mixtral-8x22B-Instruct-v0.1-GGUF), https://huggingface.co/bartowski, https://huggingface.co/mradermacher).
I tried different quants. I tried merging into one big file. And combinations. Nothing helped.

Thus, Mixtral 8x7B by far outperforms Mixtral 8x22B.

In the beginning, Mixtral 8x22B seems fine. But it is by far not production ready.
In large contrast to quantized Llama3-Models, both 7b and 70b. Works like a charme.
